run_cumulus_task.py

```diff
@@ -45,20 +51,21 @@ def run_cumulus_task(
     context_dict = vars(context) if context else {}
     logger = CumulusLogger()
     logger.setMetadata(cumulus_message, context)
-    message_adapter_disabled = os.environ.get('CUMULUS_MESSAGE_ADAPTER_DISABLED')
+    message_adapter_disabled = os.environ.get(
+        'CUMULUS_MESSAGE_ADAPTER_DISABLED', '').lower()
 
-    if message_adapter_disabled is 'true':
+    if message_adapter_disabled in ['true', 't', 'yes', 'y', 'on', '1']:
```

Comment:
To be consistent with other CMA client library, only 'true' value should be used.

Reply:
Will do.